src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+
+    private String name;
+
+    private int variable;
+
+    private int age;
+
+    private int size;
```

Comment:
```suggestion
    private String message;
    private int year;
    private int age;
    private int size;
```

bad variable names and redundant empty lines